### PR TITLE
use iron-* elements, fix bower.json, update paper-styles

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,16 +20,15 @@
...
-    "core-icon": "Polymer/core-icon#0.8-preview",
-    "core-icons": "Polymer/core-icons#0.8-preview",
-    "test-fixture": "Polymer/test-fixture#master",
-    "web-component-tester": "Polymer/web-component-tester#^2.2.0"
+    "iron-icon": "PolymerElements/iron-icon#^0.8.0",
+    "iron-icons": "PolymerElements/iron-icons#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "web-component-tester": "Polymer/web-component-tester#^2.2.0",
+    "webcomponentsjs": "Polymer/webcomponentsjs#^0.6.0"
```
Comment (cdata) [inline bower.json]: Is this the right dep? I've always used `webcomponents/webcomponentsjs`..

File: demo/index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-    Code distributed by Google as part of the polymer project is also
-    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
```
Comment (cdata) [inline demo/index.html]: License should go after `doctype`

**Comment by cdata** [comment]
Small nits, otherwise LGTM :shaved_ice:

File: bower.json (hunk 1 above)
Comment (morethanreal) [inline bower.json]: Oh, seems like the repo was moved and it redirects. I'll change to `webcomponents/webcomponentsjs`. I'm old.

File: demo/index.html (hunk 2 above)
Comment (morethanreal) [inline demo/index.html]: Will fix.

**Pull request merged**